tests/Unit/LoanServiceTest.php

```diff
@@ -64,8 +64,8 @@ public function testServiceCanCreateLoanOfForACustomer()
         ]);
         $this->assertDatabaseHas('scheduled_repayments', [
             'loan_id' => $loan->id,
-            'amount' => 1667,
-            'outstanding_amount' => 1667,
+            'amount' => 5000 - (1666 * 2),
+            'outstanding_amount' => 5000 - (1666 * 2),
```

Comment:
changing tests in such a way means changing business logic.
while it may seem _right_ from a given standpoint, it may very well break things in the system for existing users on a massive and unpredictable scale.